config/sync/core.extension.yml

```diff
@@ -187,7 +188,6 @@ module:
   twig_field_value: 0
   twig_tweak: 0
   ultimate_cron: 0
-  update: 0
```

Comment:
Looks like this is tripping config install errors. Might be worth leaving in here and removing later in another ticket, if needed.